feature/gribi/otg_tests/gribigo_compliance_test/README.md

```diff
@@ -21,3 +21,22 @@ For each compliance test case in the test suite:
     *   If the case expects a t.Fatal result, use testt.ExpectFatal.
     *   If the case expects a t.Error result, use testt.ExpectError.
     *   Otherwise, call the test case function directly.
+
+## OpenConfig Path and RPC Coverage
+
+The below yaml defines the OC paths and RPC intended to be covered by this test.
+
+```yaml
+paths:
+  /interfaces/interface/config/name:
```

Comment:
Is this path intended to be covered, or just covered because it's required for test setup? Generally, I think the pattern here has been to list the things that we are trying to fully test here, so I think this can be omitted.

@dplore -- is that right?

Reply:
Yes, that is correct.  Please only list the paths that are the focus of the test and skip listing paths.   This is of course a judgement call for the author+reviewer, but I agree this test is not intended to interface naming.  That path is the focus of tests in the [feature/interface/ folder](https://github.com/openconfig/featureprofiles/tree/main/feature/interface/), such as [feature/interface/singleton/otg_tests/singleton_test](https://github.com/openconfig/featureprofiles/tree/main/feature/interface/singleton/otg_tests/singleton_test)

Reply:
done